packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -62,8 +90,50 @@ export function ListViewItem(props) {
         }
         ref={ref}
         {...mergedProps}>
-        {item.rendered}
+        <Grid UNSAFE_className={listStyles['react-spectrum-ListViewItem-grid']} UNSAFE_style={{minHeight: '32px'}}>
```

Comment:
where does this minHeight come from?
can it be applied through css instead? i imagine it'll be different between scales